objective-c/README.md

```diff
@@ -0,0 +1,522 @@
+# Objective-c 
+
+## Introduction
+
+This document is useful if you work on a huge project and want to keep it readable, clean & maintainable.
+Also helps you to avoid common mistakes (I have found since I started my career) and shows some best-practices.
+
+## Class naming convention
+
+Use class prefixes instead of just plain names. This makes your classes more unique - imagine if everybody would use plain names: you couldn't import a third party framework without conflict.
```

Comment:
App projects typically don't need class prefixes. You are in charge of all class names and removing the prefix makes for cleaner, easier to type names (`JJWCustomerView` vs `CustomerView`). Since you control the names of the classes in your project, should any class name conflicts arise with imported libraries, you can easily adjust your class name to resolve the conflict. In my experience collisions happen rarely and the easier to read/type class names are a real benefit.

For library projects using [3-letter](http://nshipster.com/namespacing/#3rd-party-class-prefixes) class prefixes is still a very good idea.